README.md

```diff
@@ -1,304 +1,122 @@
+### Linux:
+The general format of the Cloudfuse Linux commands is `cloudfuse [command] [arguments]
 --[flag-name]=[flag-value]`
 * `help` - Help about any command
 * `mount` - Mounts a cloud storage container as a filesystem. The supported
-  containers include
-  - S3 Bucket
-  - Azure Blob Container
-  - Azure Datalake Gen2 Container
-* `mount all` - Mounts all the containers in an S3 Account or Azure account as a
-  filesystem. The supported storage services include
-  - [S3 Storage](https://aws.amazon.com/s3/)
-  - [Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)
-  - [Datalake Storage Gen2](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction)
+  containers include:
+  - [S3 Bucket Storage](https://aws.amazon.com/s3/)
+  - [Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)
+  - [Azure Datalake Storage Gen2](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction)
+  - Example: cloudfuse mount \<mount path> --config-file=\<config file>
+* `mount all` - Mounts all the containers in an S3 Account or Azure account supported by mount
+  - Example: cloudfuse mount all \<mount path> --config-file=\<config file>
 * `mount list` - Lists all Cloudfuse filesystems.
+  - cloudfuse mount list
+* `unmount` - Unmounts the Cloudfuse filesystem.
+  - Example: cloudfuse unmount \<mount path>
+* `unmount all` - Unmounts all Cloudfuse filesystems.
+  - Example: cloudfuse unmount all
+
+### Windows:
+
+The general format of the Cloudfuse Windows commands is:
+ `cloudfuse service [command] [arguments] --[flag-name]=[flag-value]`
+  * `cloudfuse service install` - Install as a Windows service
+  * `cloudfuse service uninstall` - Uninstall cloudfuse from a Windows service
+  * `cloudfuse service start` - Start the Windows service
+  * `cloudfuse service stop` - Stop the Windows service
+  * `cloudfuse service mount \<mount path>  --config-file=\<config file>` - Mount an instance that will persist in Windows when restarted
+  * `cloudfuse service unmount \<mount path>` - Unmount mount of Cloudfuse running as a Windows service
+
+### Secure options for both Windows and Linux
+To use security options the general format for cloudfuse commands is `cloudfuse [command] [arguments] --[flag-name]=[flag-value]`
 * `secure decrypt` - Decrypts a config file.
 * `secure encrypt` - Encrypts a config file.
 * `secure get` - Gets value of a config parameter from an encrypted config file.
 * `secure set` - Updates value of a config parameter.
-* `unmount` - Unmounts the Cloudfuse filesystem.
-* `unmount all` - Unmounts all Cloudfuse filesystems.
+
+Note - If you do not have admin rights, you can still mount your cloud without Windows Service, however
+the process will stay in the foreground. You may mount the cloud in the foreground using the command line arguments by removing 'service'.

```

Comment:
Since it's really only one command we're talking about, maybe it would be good to just specify it (`cloudfuse mount`) and mention that it takes the terminal so you use Ctrl+C to unmount.

Reply:
Done!